packages/jest-runtime/src/index.ts

```diff
@@ -173,6 +173,9 @@ const supportsNodeColonModulePrefixInImport = (() => {
   return stdout === 'true';
 })();
 
+const esmConditions = ['import', 'require', 'default'];
```

Comment:
Node.js doesn't pass `require` for ESM: https://nodejs.org/api/packages.html#packages_conditional_exports (`"import"` and `"require"` are mutually exclusive).

Reply:
Even though `import` supports both? I guess the idea is to fallback to `default`?

Reply:
Or just `node` I guess

Reply:
Yes, it fallbacks to either `default` or `node`.

Reply:
Should i remove it or pass `node` you think?

Reply:
I removed it for now, let's see how that works out 🙂 Better to have too little than too much